pixart.js

```diff
@@ -1 +1,20 @@
+// below is the standard code for the document ready event.
+$(document).ready(function(){
+  // jQuery goes here...
+  //  $("#inputThing").val("Hi!");
+  //  var inputText = $("input").val();
+  //  console.log(inputText);
 
+  $("#set-color").click( function(){
+    event.preventDefault();
+    var colorIn = $("input").val();
+    $(".brush").css("backgroundColor", colorIn);
+  });
+
+  for (i = 0; i < 4000; i++) {
+    $("body").append("<div class='square'></div>");
+    $(".square").mouseover( function(){
+      $(this).css("backgroundColor", $("input").val());
+    });
```

Comment:
I think this is the source of your speed issue. Instead of adding the event 4000( or 8000 times) just  append the divs in the loop and use "implicit" iteration to add the event listener. Something like this:

```js
for (i = 0; i < 4000; i++) {
  $("body").append("<div class='square'></div>");
};

$(".square").mouseover( function(){
  $(this).css("backgroundColor", $("input").val());
});
```